dmapiclient/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '25.5.0'
+__version__ = '25.5.1'
```

Comment:
It's a minor version bump so it should be:
```suggestion
__version__ = '25.6.0'
```